docker-compose.yml

```diff
@@ -4,3 +4,11 @@ services:
     build: .
     ports:
       - "20022:22"
+  puppet_5_node:
```

Comment:
I think this compose file can be setup so it only builds a single image, then starts 3 instances of it.

Reply:
https://stackoverflow.com/a/35662191/2048059 is one possibility of how to do it.